components/notebook-controller/Dockerfile

```diff
@@ -6,7 +6,7 @@
 #
 # This is necessary because the Jupyter controller now depends on
 # components/common
-ARG GOLANG_VERSION=1.12.5
+ARG GOLANG_VERSION=1.15.4
```

Comment:
minor: it would be great to keep golang version same as golang used in k8s dependency.

Reply:
@Jeffwan  Thanks, I agree with the recommendation and have updated it.

Reply:
@Jeffwan thanks for the recommendation!
Can you expand a bit on it? I have never encountered it before.
What is the reason for matching the versions of Go?